features/collections.feature

```diff
@@ -128,16 +128,18 @@ Feature: Collections
       puppies:
         output: false
     """
-    And I have a "foo.txt" file that contains "random static file"
+    And I have a "index.html" page that contains "Newest puppy: {% assign puppy = site.puppies.last %}{{ puppy.title }}"
```

Comment:
I really thought `site.puppies.first` would be the newest.

Reply:
From my experience, apparently that is not the case. The documentation doesn't seem to specify.